numba/tests/test_parallel_backend.py

```diff
@@ -549,23 +549,29 @@ def each_env_var(self, env_var: str):
 def plus(x, y):
     return x + y
 
-print(' '.join(numba.config.THREADING_LAYER_PRIORITY))
-print("@%s@" % numba.threading_layer(), file=sys.stderr)
+captured_envvar = list("{env_var}".split())
+assert numba.config.THREADING_LAYER_PRIORITY == captured_envvar, \
+    "priority mismatch"
+assert numba.threading_layer() == captured_envvar[0],\
+    "selected backend mismatch"
 """
         cmd = [
             sys.executable,
             '-c',
             code,
         ]
-        return self.run_cmd(cmd, env=env)
+        self.run_cmd(cmd, env=env)
```

Comment:
Since you remove the return statement here, please also update the docstring to match, it currently still contains the documentation for the return.

Reply:
7363275